lib/smart_proxy_remote_execution_ssh/command_action.rb

```diff
@@ -42,7 +42,7 @@ def rescue_strategy_for_self
     def command
       @command ||= Dispatcher::Command.new(:id               => input[:task_id],
                                            :host             => input[:hostname],
-                                           :ssh_user         => 'root',
+                                           :ssh_user         => input[:ssh_user] || 'root',
```

Comment:
It felt close to the private keys setting, which we want to be able to keep on per-proxy. I'm ok with letting override both remote user and private key from the server, but would still like to make the possibility to keep defaults for the private keys (and remote user that should be used) here.